doc/rst/tools/chpl-language-server/chpl-language-server.rst

```diff
@@ -0,0 +1,193 @@
+.. _readme-chpl-language-server:
+
+chpl-language-server
+====================
+
+``chpl-language-server`` (``CLS`` for short) is a language intelligence tool
+for the Chapel programming language. ``CLS`` provides many of the common
+features users expect from a modern development environment. It is built on top
+of the `Language Server Protocol (LSP)
+<https://microsoft.github.io/language-server-protocol/>`_ and is designed to be
+editor-agnostic.
+
+Getting Started
+---------------
+
+The easiest way to make ``CLS`` available is by using the
+``chpl-language-server`` Makefile target. This will build the Dyno compiler
```

Comment:
Since this is a documentation page on the website, you can no longer assume a from-source install. I would definitely include the same disclaimer as the Python bindgs: "make sure you have a from-source install, make sure you have LLVM, make sure you have `python3-dev`".

Reply:
I will solve this with a direct link to the chapel-py docs, and do the same to chplcheck